test/util.js

```diff
@@ -7,6 +7,14 @@ function redirectsTo() {
   };
 }
 
+function redirectsToAndSetsCookie() {
```

Comment:
Let's keep the test simple; the cookie functionality is provided by the _test_, not the actual code, so does not need to be tested.

Reply:
ok, so test will be quite simple then...